pydarn/plotting/rtp.py

```diff
@@ -331,7 +325,7 @@ def plot_range_time(cls, dmap_data: List[dict], *args,
 
     @classmethod
     def plot_time_series(cls, dmap_data: List[dict], *args,
-                         parameter: str = 'frequency', beam_num: int = 0,
+                         parameter: str = 'tfreq', beam_num: int = 0,
                          ax=None, time_span: tuple = None,
                          date_fmt: str = '%y/%m/%d\n %H:%M',
                          channel='all', scale: str = 'linear',
```

Comment:
Questions: what does the bit before the colon indicate? For example, `beam_num: int = 0`.  How does this differ from `beam_num=0`?

Reply:
This `: str ` is called type hinting: https://docs.python.org/3/library/typing.html

I think back in my DmapRead code I mention it but will add it to the developers guide as something we should consistently do. 

The point of type hinting is to "hint" to the user/developer what type the parameter and the return value will be since python doesn't have "types." 

I believe this a good practice to keep up with as it makes the code more readable and may be adopted into PEP 8 :p